### DISCUSSION TITLE
777

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#949] [Comments]
# 777

사내에 공유드렸던 내용을 기록차 남겨둡니다. 오늘 A Cell의 스프린트 플래닝때 이야기 드린 내용이긴한데, 글로벌 프로젝트를 진행하면서, 혹은 플레이어, 앱, (예상치못한) 데이터 엔지니어링 등 새로운 환경에 도전하는 제품팀 분들도 들으시면 좋을것 같아서 공유드려요. 저는 여자친구랑 대략 10년정도 연애를 했는데요. (2014년부터) 10년 내내 거의 안싸우다가 이번에 결혼식 준비하면서 10년치를 몰아서 싸우고 있어요. (어제도....) 10년을 만나는 동안 웬만한 커플이 싸우는 일에는 저희는 거의 싸우지 않아서 결혼준비도 무난히 잘 끝날것 같다는 생각을 했는데요. 막상 결혼식을 준비하다보니 상대가 피곤한 상황인걸 알고 있음에도 어쩔 수 없이 확답을 요구하고, 같이 움직여야하는 상황이 계속 생기더라구요..

https://jojoldu.tistory.com/777

**Comment by hojinDev** [comment]
결혼 축하드립니다.
이제야 회사를 이루게 되셨네요. ㅋㅋㅋㅋㅋ
서비스도 오픈하셔야하는데 말이죠. 물론 스타트업이 서비스 오픈없이 사회에 기여할수도 있습니다만.
트래픽도 경험해보고 장애도 맞아보시고 오픈한 서비스가 성장하는것도 보셔야죠. 
쌍둥이 추천합니다..

**Comment by mag123c** [comment]
[이 정도의 난이도가 IT 업계에서 낮은 레벨이라면 저희가 못 이겨낼 이유가 없고, 이 정도가 정말 IT 업계 전체를 봐도 높은 난이도라면 이걸 이겨내면 그 이후에 어떤 일이든 쉽게 느껴질 것이구요.]

라는 글귀가 너무 인상적이었습니다.
프로젝트 한 사이클마다 위 글귀를 새기면서 작업해야할 것 같아요 좋은 인사이트 감사합니다.
